docs/cheatsheet.rst

```diff
@@ -25,10 +26,23 @@ Global Variables
   tuple. Performs a full type-check, ensuring the types match the function signature. Result equals ``abi.encodeWithSelector(functionPointer.selector, (...))``
 - ``abi.encodeWithSignature(string memory signature, ...) returns (bytes memory)``: Equivalent
   to ``abi.encodeWithSelector(bytes4(keccak256(bytes(signature))), ...)``
+  
+Members of bytes
+================
```

Comment:
Not sure if this will work (please check the preview) but if it does, it would be better this way:
```suggestion
Members of ``bytes``
====================
```

Same for ``string`` below.

Reply:
Hii!! It's working